main/static/autocom.js

```diff
@@ -286,11 +287,9 @@ function generateTooltipContents(tooltip, words, index, selectedCallback) {
 // Inserts `word` at the location specified by `at` inside of `input`.
 function insertWord(input, at, word, supressCallback) {
 	input.value = input.value.substring(0, at.from) + word + input.value.substring(at.to);
-	setTimeout(function() {
```

Comment:
Found out that it wasn't necessary.

I can't remember why it was added originally, so I assume it was because the browser was preventing the write for some reason or another, or that it was causing an infinite loop. Either way, doesn't need t be there anymore.

You'd notice it a little, because after pressing `enter` to autocomplete, you'd see the cursor jump a little. No point in doing that, so got rid of it.